website/docs/r/log_search.html.markdown

```diff
@@ -86,9 +86,8 @@ The following arguments are supported:
 - `description` - (Optional) Description of the search.
 - `parent_id` - (Required) The identifier of the folder to create the log search in.
 - `query_string` - (Required) Log query to perform.
-- `query_parameter` - (Block List, Optional) List of query parameters for the log search. See [query parameter schema](#schema-for-query_parameter).
-    Values for search template used in the search query. 
-    Learn more about the search templates here : https://help.sumologic.com/docs/search/get-started-with-search/build-search/search-templates/
+- `query_parameter` - (Block List, Optional) Upto 10 query_parameter blocks can be added one for each parameter in the query string. 
```

Comment:
"Upto" should be "Up to".

I think you need a hyphen in here:

Up to 10 `query_parameter` blocks can be added - one for each parameter in the `query_string`.